### PR TITLE
Update active_cname.js

### PR DESCRIPTION
**Pull request by OpAayush**
- [x] 申请该域名有实际的用途
- [x] 我已经阅读并遵守限定条款

---

### CODE REVIEW

**Comment by OpAayush** [comment]
https://opaayush.github.io/Nelophine

**Comment by OpAayush** [comment]
Accept pls

**Comment by OpAayush** [comment]
https://uptimer.spicydevs.repl.co

**Comment by OpAayush** [comment]
@willin

File: main/active_cname.js [hunk 1]
```diff
@@ -119,6 +119,8 @@ module.exports = {
   'preview.spicydevs': 'spicydevs.github.io',
   'minecraft': 'mcjs.pages.dev',
   'servers': 'opaayush.github.io',
+  'nelophine': 'opaayush.github.io',
+  'uptime': '872b6773-e225-46b7-b6b9-25747d831c64.id.repl.co',
```
Comment (willin) [inline main/active_cname.js:123]: make others confusing

**Review by willin — CHANGES_REQUESTED** [review]

File: main/active_cname.js (hunk 1 above)
Comment (OpAayush) [reply main/active_cname.js:123]: Ok but pls merge fast what should I do?
All the domains i own i seperated them together

Comment (willin) [reply main/active_cname.js:123]: why not 3rd-level domains like uptime.yourname.js.cool, servers.yourname.js.cool

Comment (willin) [reply main/active_cname.js:123]: @OpAayush

**Pull request closed without merging**